crates/iroha_executor_data_model/Cargo.toml

```diff
@@ -16,5 +16,7 @@ iroha_executor_data_model_derive = { path = "../iroha_executor_data_model_derive
 iroha_data_model.workspace = true
 iroha_schema.workspace = true
 
+derive_more = { workspace = true, features = ["constructor", "from"] }
+parity-scale-codec.workspace = true
```

Comment:
```suggestion
```

why is it needed?

Reply:
Because multisig data models need to impl either `Encode` `Decode` or `WrapperTypeEncode` `WrapperTypeDecode` for `IntoSchema`

Reply:
I'm a bit confused because none of the default permissions in `executor_data_model/src/permission.rs` implement `Decode`/`Encode`

Reply:
That's probably because the permissions are not listed in [types!](https://github.com/hyperledger-iroha/iroha/blob/5677cf1bc87164cef8b6a216ee88b45921b5e01f/crates/iroha_schema_gen/src/lib.rs#L107)

Reply:
can't you do the same for custom instructions?